### PR TITLE
Unruly: Protected Audience Support

### PR DESCRIPTION
**Pull request by bhainesnexxen**
## Type of change
- [X] Feature
- [X] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change
Adds support for Protected Audiences.

## Other information
Documentation PR: https://github.com/prebid/prebid.github.io/pull/4977

### CODE REVIEW

File: test/spec/modules/unrulyBidAdapter_spec.js [hunk 1]
```diff
@@ -42,9 +42,40 @@ describe('UnrulyAdapter', function () {
     }
   }
 
-  const createExchangeResponse = (...bids) => ({
-    body: {bids}
-  });
+  function createOutStreamExchangeAuctionConfig() {
+    return {
+      'seller': 'https://targeting.unrulymedia.com',
+      'decisionLogicURL': 'https://targeting.unrulymedia.com/padecisionlogic',
```
Comment (patmmccann) [inline test/spec/modules/unrulyBidAdapter_spec.js:48]: this doesnt exists

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by bhainesnexxen** [comment]
We submitted this PR in anticipation of our backend becoming available within a week. Upon further consideration, we'll re-submit after the entire pipeline's available.

**Comment by bhainesnexxen** [comment]
Testing URLs are live now.

File: test/spec/modules/unrulyBidAdapter_spec.js (hunk 1 above)
Comment (bhainesnexxen) [reply test/spec/modules/unrulyBidAdapter_spec.js]: The URL's updated and live. Note that this was just for testing purposes and not used in the production adapter.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**